### PR TITLE
UML-711-cancel-access-code-button

### PR DESCRIPTION
**Pull request by nickjones-moj**
Changed button to link for 'Cancel organisation's access'

Fixes UML-711

## Checklist

* [x] I have performed a self-review of my own code
* [ ] I have added relevant logging with appropriate levels to my code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [ ] I have added tests to prove my work
* [x] The product team have tested these changes

### CODE REVIEW

File: service-front/app/src/Actor/templates/actor/partials/cancel-code-option.html.twig [hunk 1]
```diff
@@ -8,7 +8,8 @@
                         {{ govuk_form_element(code.form.get('viewer_code')) }}
                         {{ govuk_form_element(code.form.get('organisation')) }}
 
-                        <button data-prevent-double-click="true" type="submit" class="govuk-button govuk-button--warning">Cancel organisation's access</button>
+                  <!--      <button data-prevent-double-click="true" type="submit" class="govuk-button govuk-button--warning">Cancel organisation's access</button> -->
```
Comment (cooperaj) [inline service-front/app/src/Actor/templates/actor/partials/cancel-code-option.html.twig:11]: No need for the commented code - thats what git is for :)

File: service-front/web/src/scss/_lpa-typography.scss [hunk 2]
```diff
@@ -10,3 +10,11 @@
 .lpa-secondary-text-colour {
   color: #626a6e;
 }
+
+button.org-cancel {
+  background: none!important;
+  border: none;
+  padding: 0!important;
```
Comment (GemTay) [inline service-front/web/src/scss/_lpa-typography.scss:17]: Are both of the !important's necessary here? We should avoid using that where we can

Comment (mattmachell) [reply service-front/web/src/scss/_lpa-typography.scss:17]: You can probably fix this more cleanly by adjusting the order of declaration or adding specificity.

File: service-front/app/src/Actor/templates/actor/partials/cancel-code-option.html.twig (hunk 1 above)
Comment (nickjones-moj) [reply service-front/app/src/Actor/templates/actor/partials/cancel-code-option.html.twig]: removed

File: service-front/web/src/scss/_lpa-typography.scss (hunk 2 above)
Comment (nickjones-moj) [reply service-front/web/src/scss/_lpa-typography.scss]: Removed them both - not required. Left overs from when I was implementing.

**Pull request merged**